aio/aio-proxy/aio_proxy/response/formatters/complements.py

```diff
@@ -0,0 +1,41 @@
+from aio_proxy.response.unite_legale_model import UniteLegaleComplements
+
+
+def format_complements(
```

Comment:
Something does not really work with this function. A lot of text for a wrapper that only call `UniteLegaleComplements`

Reply:
It is called `format_complements`but I cannot find ant formatting action in it. Just mapping